src/main/java/com/vmtapp/enterprise/dao/TicketSQLDAO.java

```diff
@@ -24,6 +24,11 @@
     @Autowired
     TicketRepository ticketRepository;
 
+    /**
+     * Takes a ticket as a parameter to be saved
+     * @param ticket a ticket object to be saved
+     * @return return the saved ticket
+     */
```

Comment:
Only add JavaDoc if it is providing value over and above what can be read from the method.